ui/app/components/app/gas-customization/gas-price-button-group/index.scss

```diff
@@ -87,8 +87,8 @@
   }
 
   &__primary-currency {
-    font-size: 12px;
-    line-height: 12px;
```

Comment:
These two changes to `line-height` do make the inline gas buttons in the send flow look a bit crowded. The smaller line-height was relied upon to ensure there was padding, so now the text is pressed up against the border.

Before:

![before-inline-send](https://user-images.githubusercontent.com/2459287/89461593-8f499900-d742-11ea-8389-9d32ad92fd62.png)

After:

![after-inline-send](https://user-images.githubusercontent.com/2459287/89461568-8953b800-d742-11ea-941d-ede55f87f6d8.png)

Maybe we could restore the old `line-height` for now? Or make the row a few pixels taller to give it more room.

Reply:
I resolved this by changing the padding of the buttons from `0 4px` to `4px`, moving the Advanced Options button into its own SendRow instead of being inside the Transaction Fee Row, so that I can align-items: center on the rows so that the labels are centered with their row contents

Reply:
![image](https://user-images.githubusercontent.com/4448075/90808612-ebd1ba00-e2e5-11ea-8976-e7a9977952ef.png)

![image](https://user-images.githubusercontent.com/4448075/90808678-086df200-e2e6-11ea-9f07-71f68cae2227.png)